src/_nebari/provider/cicd/github.py

```diff
@@ -226,9 +216,6 @@ def gen_nebari_ops(config):
     step3 = install_nebari_step(config.nebari_version)
     gha_steps = [step1, step2, step3]
 
-    if config.provider == schema.ProviderEnum.gcp:
-        gha_steps.append(setup_gcloud())
```

Comment:
If this is no longer needed, it seems like it wouldn't be needed in https://github.com/nebari-dev/nebari/blob/aed2b926fc1176fa90e4b23e6ee9dc02ca0e861d/.github/workflows/test_gcp_integration.yaml#L73 anymore either.  Is that right?

Reply:
@Adam-D-Lewis Upon further research, I think we actually need it so we can authenticate to Google Cloud and retrieve the credentials to set the `GOOGLE_CREDENTIALS` environment variable.

The workflows for the other cloud providers also do this. See:
* https://github.com/nebari-dev/nebari/blob/aed2b926fc1176fa90e4b23e6ee9dc02ca0e861d/.github/workflows/test_azure_integration.yaml#L73-L78
* https://github.com/nebari-dev/nebari/blob/aed2b926fc1176fa90e4b23e6ee9dc02ca0e861d/.github/workflows/test_aws_integration.yaml#L72-L77

This makes sense in that we're not setting the `gcloud` CLI but just getting the proper Google Cloud credentials.